doc/btp_ccp.txt

```diff
@@ -0,0 +1,191 @@
+CCP Service (ID 19)
+===================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<non-controller>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02: CCP Register Callbacks
```

Comment:
I'm not sure about this command.  This looks like some implementation detail of zephyr stack exposed over BTP...

Can you explain why is this needed?

Reply:
Yes - This is part of the essential setup of the CCP.  It is used to register all callbacks from both discovery and individual call control commands.  Without it - there is no CCP functionality.
You could argue that it could have been called internally by including a check in each command, but that would just mean a lot of extra housekeeping.

Reply:
but this is zephyr API specific, if you need to register any callbacks in tester app just do it on service init

Reply:
Yes - sure that is possible.  I will do that.

Reply:
Done.